sirepo/pkcli/service.py

```diff
@@ -45,10 +46,13 @@ def cfg():
 
 def flask():
     from sirepo import server
+    import sirepo.pkcli.setup_dev
 
     with pkio.save_chdir(_run_dir()):
-        use_reloader = pkconfig.channel_in('dev')
-        app = server.init(use_reloader=use_reloader)
+        sirepo.pkcli.setup_dev.default_command()
+        # above will throw better assertion, but just in case
+        assert pkconfig.channel_in('dev')
+        app = server.init(use_reloader=True)
```

Comment:
`u=True` and then use u here and on line 64.

Reply:
In this case, I don't think the abstraction adds value. It's a symbolic substitution for True, but the word True speaks for itself, and there are only two lines copied in the same module. I think this is where Go's "a little copying" applies. If it were in two functions, the explicit coupling would be valuable, but in this case, I think it's better to copy.